packages/synapse-react-client/src/lib/containers/download_list_v2/CreatePackageV2.tsx

```diff
@@ -71,13 +71,13 @@ export const CreatePackageV2 = (props: CreatePackageV2Props) => {
     }
   }
 
-  const onChange = (event: React.SyntheticEvent<HTMLInputElement>) => {
-    setZipFileName(event.currentTarget.value)
+  const onChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setZipFileName(event.target.value)
   }
 
   return (
     <>
-      <div className="CreatePackageV2 bootstrap-4-backport">
```

Comment:
Remove bootstrap-4-backport from the parent -- it breaks the MUI TextField styling